aws-roles/outputs.tf

```diff
@@ -0,0 +1,14 @@
+data "aws_caller_identity" "current" {}
+
+locals {
+  spinnaccount = "export AWS_ACCOUNT_NAME=aws-dev-1"
+  account = "export ACCOUNT_ID=${data.aws_caller_identity.current.account_id}"
+  rolename = "export ROLE_NAME=role/${aws_iam_role.managedRole.name}"
+  halconfig = "hal config provider aws account add $AWS_ACCOUNT_NAME --account-id $ACCOUNT_ID --assume-role $ROLE_NAME"
+  halenable = "hal config provider aws enable"
+  haldeploy = "hal deploy apply"
+}
+
+output "commands" {
```

Comment:
for aws accounts to work, a list of region or a single region need to be provided, we need to set this output to have a region. 
Output needs to account for this and we need to tell the user to set it's region.